src/main/java/net/snowflake/client/jdbc/SnowflakeSQLLoggedException.java

```diff
@@ -50,14 +45,17 @@
    * @param value JSONnode containing relevant information specific to the exception constructor
    *     that should be included in the telemetry data, such as sqlState or vendorCode
    * @param ex The exception being thrown
+   * @param oobInstance Out of band telemetry instance through which log will be passed
    */
-  private void sendOutOfBandTelemetryMessage(JSONObject value, SnowflakeSQLLoggedException ex) {
+  private static void sendOutOfBandTelemetryMessage(
```

Comment:
These telemetry-sending messages are now static so that they can be called by the newly-created static function logSqlFeatureNotSupportedException().